salt/metalk8s/kubelet/installed.sls

```diff
@@ -11,6 +11,22 @@ Install kubelet dependencies:
       - socat
       - conntrack-tools
 
+Install and configure cri-tools:
+  pkg.installed:
+    - name: cri-tools
+    - fromrepo: {{ repo.kubernetes.name }}
+    - require:
+      - pkgrepo: Configure Kubernetes repository
+  file.serialize:
+    - name: /etc/crictl.yaml
```

Comment:
Maybe we can add the merge_if_exists arg so that any custom option in crictl.yaml would not be overwritten

Reply:
Good point

Reply:
Done